run_snafu.py

```diff
@@ -35,7 +35,12 @@
 urllib3_log = logging.getLogger("urllib3")
 urllib3_log.setLevel(logging.CRITICAL)
 
-setup_loggers("snafu", logging.DEBUG)
+if os.getenv("snafu_debug") != None:
+    setup_loggers("snafu", logging.DEBUG)
+    logger.info("logging level is DEBUG")
+else:
+    setup_loggers("snafu", logging.INFO)
```

Comment:
and then this can be 

```
setup_loggers("snafu", args.loglevel) 
```

Reply:
+1

Reply:
that's ok with me, my point is that there should be SOME way other than code change to have run_snafu.py turn on display of documents that it's pushing to ES.    And I'll add change to documentation to explain how to do this.